install.sh

```diff
@@ -275,30 +286,36 @@ if [[ "$choice" =~ ^(yes|y|Y) ]]; then
     read -p "Enter the database password: " databasePassword
     read -p "Enter the database name: " databaseName
 
-    userExists=$(sudo -u postgres psql -c "SELECT * FROM pg_user WHERE username = '${databaseUsername}'" | grep -c "1 row")
+    userExists=$(sudo -i -u postgres psql -c "SELECT * FROM pg_user WHERE usename = '${databaseUsername}'" | grep -c "1 row")
+    databaseExists=$(sudo -i -u postgres psql -l | grep "${databaseName}" | wc -l)
 
     if [[ $userExists == 1 ]]; then
         read -p "The database user ${databaseUsername} already exists, do you want to overwrite it? [y/N]: " choice
 
         if [[ "$choice" =~ ^(yes|y|Y) ]]; then
```

Comment:
`yes` is redundant because the single `y` will match it. I.e. `^(yes|y|Y)` is identical to `^(y|Y)`. Same for `^(no|n|N)` below.